README.md

```diff
@@ -109,6 +109,10 @@ For example:
 
 Arguments before the `--` are interpreted by the JVM while arguments after `--` are passed to Metrics Portal.
 
+##### Extras
```

Comment:
I think rename this section to Reporting and please mention if using the published Docker image that this is available by default and how one configures the chromium path to use it.